testing/trino-tests/src/test/java/io/trino/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -5131,6 +5131,29 @@ public void testShowFunctions()
         assertFalse(functions.containsKey("like"), "Expected function names " + functions + " not to contain 'like'");
     }
 
+    @Test
+    public void testLargePivot()
+    {
+        int numberColumns = 254;
```

Comment:
Add a test what happens with 255 columns so that we are forced to update the test when the limitation is shifted. This is necessary for the test to stay relevant.

Reply:
i.e. what is the failure with `numberColumns+1` query